### ISSUE TITLE
[BUG] ChromeDriver binary "\vendor\bin\chromedriver" does not exist on Windows

### ISSUE THREAD

**Issue by w33zy** [lucatume/wp-browser#647]
**Environment**
OS: Windows 11
PHP version: 8.2.0 
Installed Codeception version: 5.0.11
Installed wp-browser version: 4.0.4  
WordPress version: 6.4-alpha-56665  
Local development environment: Laragon 6.0
WordPress structure and management: default

**Can you perform the test manually?**
No.

**Codeception configuration file**  
```yaml
namespace: Tests
support_namespace: Support
paths:
    tests: tests
    output: tests/_output
    data: tests/Support/Data
    support: tests/Support
    envs: tests/_envs
actor_suffix: Tester
params:
    - tests/.env
extensions:
    enabled:
        - Codeception\Extension\RunFailed
    config: {  }
    commands:
        - lucatume\WPBrowser\Command\RunOriginal
        - lucatume\WPBrowser\Command\RunAll
        - lucatume\WPBrowser\Command\GenerateWPUnit
        - lucatume\WPBrowser\Command\DbExport
        - lucatume\WPBrowser\Command\DbImport
```

**Env file**
```env
WORDPRESS_ROOT_DIR=D:\Laragon\www\collections
WORDPRESS_DB_URL=mysql://root:@localhost:3306/collections
TEST_TABLE_PREFIX=test_
WORDPRESS_TABLE_PREFIX=wp_
WORDPRESS_URL=https://collections.local
WORDPRESS_DOMAIN=collections.local
WORDPRESS_ADMIN_USER=wzy
WORDPRESS_ADMIN_PASSWORD=uswu44241
CHROMEDRIVER_HOST=localhost
CHROMEDRIVER_PORT=9515
```

**Suite configuration file**
```yaml
# Integration suite configuration
#
# Run integration and "WordPress unit" tests.

actor: EndToEndTester
bootstrap: _bootstrap.php
modules:
    enabled:
        - lucatume\WPBrowser\Module\WPWebDriver
        - lucatume\WPBrowser\Module\WPDb
        - lucatume\WPBrowser\Module\WPFilesystem
        - lucatume\WPBrowser\Module\WPLoader
    config:
        lucatume\WPBrowser\Module\WPWebDriver:
            url: '%WORDPRESS_URL%'
            adminUsername: '%WORDPRESS_ADMIN_USER%'
            adminPassword: '%WORDPRESS_ADMIN_PASSWORD%'
            adminPath: '/wp-admin'
            browser: chrome
            host: '%CHROMEDRIVER_HOST%'
            port: '%CHROMEDRIVER_PORT%'
            window_size: 1200x1000
            capabilities:
              "goog:chromeOptions":
                args:
                  - "--headless"
                  - "--disable-gpu"
                  - "--disable-dev-shm-usage"
                  - "--proxy-server='direct://'"
                  - "--proxy-bypass-list=*"
                  - "--no-sandbox"
        lucatume\WPBrowser\Module\WPDb:
            dbUrl: '%WORDPRESS_DB_URL%'
            dump: 'tests/Support/Data/dump.sql'
            populate: true
            cleanup: true
            reconnect: false
            url: '%WORDPRESS_URL%'
            urlReplacement: false
            tablePrefix: '%WORDPRESS_TABLE_PREFIX%'
        lucatume\WPBrowser\Module\WPFilesystem:
            wpRootFolder: '%WORDPRESS_ROOT_DIR%'
        lucatume\WPBrowser\Module\WPLoader:
            loadOnly: true
            wpRootFolder: "%WORDPRESS_ROOT_DIR%" 
            dbUrl: '%WORDPRESS_DB_URL%'
            domain: '%WORDPRESS_DOMAIN%'
            
# Integration suite configuration
#
# Run integration and "WordPress unit" tests.

actor: IntegrationTester
bootstrap: _bootstrap.php
modules:
    enabled:
        - lucatume\WPBrowser\Module\WPLoader
    config:
        lucatume\WPBrowser\Module\WPLoader:
           wpRootFolder: "%WORDPRESS_ROOT_DIR%" 
           dbUrl: '%WORDPRESS_DB_URL%'
           wpDebug: true
           tablePrefix: '%TEST_TABLE_PREFIX%'
           domain: '%WORDPRESS_DOMAIN%'
           adminEmail: 'admin@%WORDPRESS_DOMAIN%'
           title: 'Integration Tests'
           plugins: ['test-plugin-3/test-plugin-3.php']
           theme: ''
```

**Describe the bug**
On a fresh installation of `wp-browser` run running the `vendor/bin/codecept run Integration` command as per the documentation, I get the follow error in my terminal window.

**Output**
```sh
Codeception PHP Testing Framework v5.0.11 https://helpukrainewin.org
Starting ChromeDriver on port 52744 ...
In ChromeDriver.php line 32:

  ChromeDriver binary D:\Laragon\www\wpbrowser\wp-content\plugins\test-plugin-3\vendor\bin\chromedriver does not exist.
```

**To Reproduce**
Follow setup instruction on Window 11 for default configuration

**Expected behavior**
No errors

**Comment by w33zy** [comment]
I would like to add that this will always fail on Windows machine because the check on https://github.com/lucatume/wp-browser/blob/7adff1c7f7d3b5a6df7d702d4c904a40b327e7a4/src/ManagedProcess/ChromeDriver.php#L31 needs to look for `chromedriver.exe`.

So this line could be changed to:
```php
if (!(
    (file_exists($chromeDriverBinary) && is_executable($chromeDriverBinary)) ||
    (file_exists($chromeDriverBinary . '.exe') && is_executable($chromeDriverBinary . '.exe'))
)) {
    throw new RuntimeException(
        "ChromeDriver binary $chromeDriverBinary does not exist.",
        ManagedProcessInterface::ERR_BINARY_NOT_FOUND
    );
}
```

**Comment by lucatume** [comment]
Fixed in version `4.0.5`; should this not be the case, let me know.
